src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  --main-color: #00acdc;
+}
+
+header {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  min-width: 1024px;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  padding: 10px 0;
+  box-sizing: initial;
```

Comment:
same, you don't need initial box-sizing now, cause you don't have padding or border on the element
```suggestion
```